node_modules/oae-util/lib/server.js

```diff
@@ -86,6 +86,16 @@ var setupServer = module.exports.setupServer = function(port, _config) {
         return next();
     });
 
+    // Trim whitespace on all post parameters
+    app.use(function(req, res, next) {
+        _.each(req.body, function(value, key) {
+            if (_.isString(value)){
+                req.body[key] = value.trim();
+            }
+        });
+        return next();
+    });
```

Comment:
So, my first thought was to have the tests register a new route that would just echo back the body after it goes through all the middleware, then I could post a bunch of fields and verify that they all transform properly. That doesn't appear to work though, I think because there's a finalize step for the routes that runs before the tests. So, I'm open to suggestions on better ways to test this (or getting the route to register from a test).